ChangeHistory

```diff
@@ -6,6 +6,9 @@ Releases are tagged on the 'master' branch as "g4cmp-Vxx-yy-zz".
 Features and bug fixes are tagged on the 'develop' branch using the JIRA
 ticket identifier, "G4CMP-nnn".
 
+2023-08-31  G4CMP-381 : Compute rotation matrices from valley's direction
+			instead of Euler angles. Add anti-valleys.
```

Comment:
Do you think you could shorten this to one line?  If not, it's fine as it is.

Reply:
Changed the comment to : Add valleys and anti-valleys from Miller indices.

Tell me if it's okay!